src/repo-gulpfile.js

```diff
@@ -153,6 +156,71 @@ const generateVersioningFile = async function () {
     )
 }
 
+
+const generateHomepage = async function () {
+    /* 
+     * If a repository.json is defined at the root of the repository, this function generate a homepage for the repository.
+     * It will be based on repository.json and versioning.json data and homepage.pug file.
+     * 
+     * See homepage.pug for more information on how to create repository.json file
+     */
+
+    //joining path of directory
+    let basePath = process.cwd().substr(0, process.cwd().length - 44)
+    let repositoryDataPath = path.join(basePath, 'repository.json')
+    let versioningFilePath  = path.join(basePath, 'bundles/versioning.json')
+
+    const directoryPath = path.join(basePath, 'src')
+    var promises = []
```

Comment:
```suggestion

```
Unused value

Reply:
Woups... I forgot to delete this line. Thanks